wnt/protocol_buffers_files/message.proto

```diff
@@ -293,22 +295,42 @@ message DiagnosticsData {
     // New in 5.1 continues
     optional int32 cca_limit_dBm = 101;
 
-    // New in WM 5.2
+    // New in 5.2
     optional uint32 address_clashes = 102;
+    optional uint32 ll_device_count = 103;
+
+    // New in 5.3
+    optional bool is_ble_scan_active = 104;
+    optional uint32 cost_compensation_throughput = 105;
```

Comment:
I know these and terms come from the stack diagnostics and terminology used there... But commenting still 😁.

Anyway, the "compensation" tends to have "positive" effect on the end result, whereas these are more like penalties. So in a sense these are more like `route_cost_penalty_xxx`.

But as said, current terminology is fine also.